http-generator-core/src/main/java/io/avaje/http/generator/core/ElementReader.java

```diff
@@ -249,22 +252,14 @@ void writeParamName(Append writer) {
    * Build the OpenAPI documentation for this parameter.
    */
   void buildApiDocumentation(MethodDocBuilder methodDoc) {
-    if (!isPlatformContext() && !isParamMap && paramType != ParamType.BEANPARAM) {
-      if (includeParam()) {
-        new MethodParamDocBuilder(methodDoc, this).build();
-      }
+    if (!isPlatformContext()
+        && !isParamMap
+        && paramType != ParamType.BEANPARAM
+        && !IgnorePrism.isPresent(element)) {
+      new MethodParamDocBuilder(methodDoc, this).build();
     }
   }
 
-  private boolean includeParam() {
```

Comment:
Pretty sure that removing this means that the generated openapi includes the "ignored" field.  Pretty sure we need to put this back.

I'll look to merge this in and confirm that we need to put this part back in.

Reply:
I added a ` && !IgnorePrism.isPresent(element)` condition using the ignore prism